content/md/articles/cookbooks/cli_build_projects.md

```diff
@@ -772,3 +772,48 @@ and calls `jar` which uses
 [both `with-dir` and `with-project-root`](https://github.com/polyfy/polylith/blob/9053b190d5f3b0680ac4fe5c5f1851f7c0d40830/build.clj#L146-L147)
 to set the project root while performing `tools.deps` and `tools.build`
 operations.
+
+## Including java code in a Clojure project
+Many Clojure projects will, at some point, require the inclusion of Java files.
```

Comment:
We have 140k lines of Clojure at work and no Java so I would challenge that assertion. I think something like this wording would be better:

Some Clojure projects may decide to implement some functionality directly in Java, which then requires a compilation step for those Java files.

Reply:
Sure!